### PR TITLE
update certbot cron job to request certificate for www.cantusdatabase.org

### LINKED ISSUE #1020: no www certificate?
It seems our certificate is only valid for "cantusdatabase.org": adding a www leads to a warning. I think we want the address to be valid with or without the www (we reference it both ways in different places).
image

### PR DESCRIPTION
**Pull request by jacobdgm**
...as well as just cantusdatabase.org

should fix #1020.

### CODE REVIEW

File: cron/cron.txt [hunk 1]
```diff
@@ -10,4 +10,4 @@
 20      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/monthly.sh
 30      4       1       1       *       bash /home/ubuntu/code/CantusDB/cron/postgres/yearly.sh
 40      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_next_chant_fields; bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_is_last_chant_in_feast
-50      4       *       *       0       /usr/local/bin/docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org --renew-hook "nginx -s reload"
+50      4       *       *       0       /usr/local/bin/docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org www.cantusdatabase.org --renew-hook "nginx -s reload"
```
Comment (dchiller) [inline cron/cron.txt]: I don't know anything about certbot, so I don't know that my approval should mean anything! Did it work? When I quickly checked the docs, it looks like they pass multiple domains differently than you do here: 

https://eff-certbot.readthedocs.io/en/stable/using.html#certbot-commands

**Comment by jacobdgm** [comment]
so, some interesting output when I tried to run the updated command with multiple -d flags:

```
jacob@cantus-db-prod:~$ /usr/local/bin/docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org -d www.cantusdatabase.org --renew-hook "nginx -s reload"
ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
        
jacob@cantus-db-prod:~$ cd ../ubuntu/code/CantusDB/
jacob@cantus-db-prod:/home/ubuntu/code/CantusDB$ /usr/local/bin/docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org -d www.cantusdatabase.org --renew-hook "nginx -s reload"
Unable to find renew-hook command nginx in the PATH.
(PATH is /usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
See also the --disable-hook-validation option.
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /tmp/certbot-log-q6811rm2/log or re-run Certbot with -v for more details.
jacob@cantus-db-prod:/home/ubuntu/code/CantusDB$ docker-compose exec nginx bash
root@8fbbd1cd51f4:/# nginx -s reload
2023/09/13 22:43:30 [notice] 35#35: signal process started
```
the above didn't lead to any change in the certificate displayed in the browser. Neither did restarting the containers.

**Comment by jacobdgm** [comment]
> When I quickly checked the docs, it looks like they pass multiple domains differently than you do here...

You're absolutely right - I misunderstood a comment @kqct made on #870. I've updated this.

**Comment by jacobdgm** [comment]
update: running `docker-compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ -d cantusdatabase.org -d www.cantusdatabase.org` and then restarting the containers worked - our certificates (both for the domain with "www" and without) are now good until mid-December. So the two `-d` flags work.

**Review by dchiller — APPROVED** [review]
With the caveat that this stuff is very unfamiliar, it looks good and it works, so approved in my book...

**Pull request merged**